From 81e21b5b95fa2c5cb504fa9106934f8d86c46bc5 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <j@w1.fi>
Date: Fri, 27 Dec 2024 22:48:13 +0200
Subject: [PATCH] tests: Run mbo_assoc_disallow in the beginning of UML VM

This test case could take significant amount of time in some cases when
run after large number of other test cases with UML. This might be due
to multiple tshark operations that seemed to suffer from some resource
issues with UML time travel.

Signed-off-by: Jouni Malinen <j@w1.fi>
---
 tests/hwsim/vm/parallel-vm.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/hwsim/vm/parallel-vm.py b/tests/hwsim/vm/parallel-vm.py
index e504c94f7..0d342a384 100755
--- a/tests/hwsim/vm/parallel-vm.py
+++ b/tests/hwsim/vm/parallel-vm.py
@@ -103,6 +103,7 @@ uml_issue_tests = ["eht_connect_invalid_link",
                    "p2p_channel_random_social_with_op_class_change",
                    "ap_open_ps_mc_buf",
                    "wmediumd_path_rann",
+                   "mbo_assoc_disallow",
                    "ieee8021x_reauth_wep"]
 
 # Test cases that depend on dumping full process memory
-- 
2.43.0

